smoketest/graphql2rotations_test.go

```diff
@@ -0,0 +1,153 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/target/goalert/smoketest/harness"
+	"testing"
+)
+
+func TestGraphQL2RotationsFavorite(t *testing.T) {
+	t.Parallel()
+	sql := `
+	insert into users (id, name, email, role) 
+	values 
+		({{uuid "user"}}, 'bob', 'joe', 'admin');
+
+	insert into rotations (id, name, description, type, start_time, time_zone)
+	values
+		({{uuid "r1"}}, 'test', 'test', 'daily', now(), 'UTC'),
+		({{uuid "r2"}}, 'test2', 'test2', 'daily', now(), 'UTC');
```

Comment:
We can get rid of the extra rotation since there's no functional difference between the two.

It should be enough to test setting & unsetting favorite on a single rotation.